Scripts/Setup/setupExperiment.sh

```diff
@@ -404,7 +407,7 @@ if [[ "${DATATYPE}" == 'CESM' ]] || [[ "${DATATYPE}" == 'CCSM' ]] || [[ "${DATAT
 elif [[ "${DATATYPE}" == 'CFSR' ]]; then
   ln -sf "${WPSSRC}/ungrib/Variable_Tables/${VTABLE_PLEV}" 'Vtable.CFSR_plev'
   ln -sf "${WPSSRC}/ungrib/Variable_Tables/${VTABLE_SRFC}" 'Vtable.CFSR_srfc'
-elif [[ "${DATATYPE}" == 'ERA-I' ]]; then
+elif [[ "${DATATYPE}" == 'ERA-I' ]] || [[ "${DATATYPE}" == 'ERA5' ]]; then
```

Comment:
actually I think this should just be ``else``, otherwise NARR is omitted, but I think as long as none of the other conditions are met and VTABLE is defined, this is the right course of action; alternatively, check if VTABLE is defined and raise an error if not...

Reply:
Done.